docs/core.rst

```diff
@@ -69,6 +69,16 @@ Context
     :members:
     :exclude-members: tank
 
+Pipeline Configuration Utilities
+================================
+
+It is possible to enquiry about the location of various components of the pipeline configuration and
+the currently running version of the Toolkit Core.
+
+.. autofunction:: get_python_interpreter_for_config
```

Comment:
However, I would argue that having it be part of the public API would give some pause to anyone who wants to get rid of the code in the future.